sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -1013,9 +1013,15 @@ Page<AnomalyIncident> NextPageFunc(string nextLink, int? pageSizeHint)
             return PageableHelpers.CreateEnumerable(FirstPageFunc, NextPageFunc);
         }
 
-        // TODODOCS.
         /// <summary>
+        /// Gets the automatic suggestions for likely root causes of an incident.
```

Comment:
what does automatic suggestions means? what will be a "manual" suggestion?

Reply:
I guess all suggestions are automatic :)